packages/t/tailscale/files/defaults.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/cmd/tailscaled/tailscaled.service b/cmd/tailscaled/tailscaled.service
+index d5a65a4..2a3825a 100644
+--- a/cmd/tailscaled/tailscaled.service
++++ b/cmd/tailscaled/tailscaled.service
+@@ -5,7 +5,7 @@ Wants=network-pre.target
+ After=network-pre.target NetworkManager.service systemd-resolved.service
+ 
+ [Service]
+-EnvironmentFile=/etc/default/tailscaled
++EnvironmentFile=/usr/share/defaults/tailscale/tailscaled
```

Comment:
What about using something like this so it can be easily overridden?

```
EnvironmentFile=/usr/share/defaults/tailscale/tailscaled
EnvironmentFile=-/etc/default/tailscaled
```